examples/with-linaria/next.config.js

```diff
@@ -0,0 +1,19 @@
+const withCSS = require('@zeit/next-css')
+
+module.exports = withCSS({
+  webpack (config, options) {
+    const jsRule = config.module.rules.findIndex(loader => loader.test.test('test.jsx'))
```

Comment:
What does this look for? 🤔

Reply:
This is a dirty hack, but I haven’t found another solution. I would appreciate if you help.

Reply:
But what are you actually looking for? 😄 The babel loader?

Reply:
Yes, I'm looking for him to add linaria-loader